isle/FAQ.md

```diff
@@ -1,5 +1,66 @@
 # Frequently Asked Questions
 
+## Day 1: launching `test-easter-1`
+
+### What does ‘Top N’ mean?
+It means that only the top N% of the validator set will be automatically included in the validator set of the consumer chain and required to run it (otherwise they will be slashed). This is similar to how Neutron and Stride (using Replicated Security with a 5% soft opt-out) require 95% of the Hub’s validator set to run those consumer chains or face slashing penalties. The remaining 5% of the set can choose to validate if they wish, but are not required to do so.
+
+### How do I calculate if I’m in the top N or not?
+The percentage in Top N refers to the voting power percentage, not the number of validators. 
+
+So if there are 180 validators in the active set on the Hub and we have a Top N=80% chain, you would check to see if your voting power is in the top 80% of the active set. 
+
+It does NOT refer to the rank! This is important!
+
+### Can chains change their N number?
+Not right now! It’s on the roadmap for ICS 2.0 though.
+
+The logic for setting the N parameter for a consumer chain occurs on the provider side so it could be adjusted via a state migration on the provider if absolutely necessary.
+
+### What different ways are there to opt-in to a consumer chain with an assigned key?
+There are two ways to opt-in to a consumer chain using an assigned key:
+Submit the assign-consensus-key transaction and the opt-in transaction separately, without specifying a pubkey in the opt-in transaction.
+Submit the opt-in transaction using the pubkey as an argument.
```

Comment:
```suggestion
There are two ways to opt-in to a consumer chain using an assigned key:
* Submit the assign-consensus-key transaction and the opt-in transaction separately, without specifying a pubkey in the opt-in transaction.
* Submit the opt-in transaction using the pubkey as an argument.
```
It will look better when the markdown is rendered